docs/deployment/connect-scm.md

```diff
@@ -52,11 +52,11 @@ If you opted to scan a GitHub or GitLab repository when you initially signed in,
 1. Optional: If you have created more than one Semgrep account, select the account you want to make a connection for by clicking on the **Navigation bar > Your account name > The account you want to connect**.<br />
    <img src="/docs/img/more-accounts-dropdown.png" height="350px" />
 1. Go to **<i class="fa-solid fa-gear"></i> Settings > Source code managers > Add > Azure DevOps**. 
-1. In the **Connect your Azure DevOps organization** dialog box, provide:
-   - The **Name of your Azure DevOps organization**
+1. In the **Connect your Azure DevOps Project** dialog box, provide:
+   - The **Name of your Azure DevOps Project**. Ensure that the value you provide follows the format `{organization}/{project}`. This information can be seen in the project URL, for example `https://dev.azure.com/{organization}/{project}`. 
```

Comment:
```suggestion
   - The **Name of your Azure DevOps Project**. Ensure that the value you provide follows the format `{organization}/{project}`. This information can be seen in the project URL, for example `https://dev.azure.com/{organization}/{project}`. 
```
should the {organization}/{project} be placeholders? note that you can now format using `<PL>` if you include the placeholder component? grep/search for `<PL>` in the docs folder for an example.